docker-compose.yml

```
@@ -0,0 +1,48 @@
+version: '3'
+services:
+
+  db:
+    image: postgres:10-alpine
```

Not sure what version we need, but 10 is pretty old. Switched to 12
```suggestion
    image: postgres:12
```